src/main/java/Baymax.java

```diff
@@ -0,0 +1,92 @@
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Baymax {
+    public static void main(String[] args) {
+        String name = "Baymax";
+        System.out.println("Hello! I'm " + name + ", your personal task manager." + System.lineSeparator()
+                + "What can I do for you today?" + System.lineSeparator() + System.lineSeparator());
+
+        ArrayList<Task> taskArrayList = new ArrayList<>();
+        int index = 0;
+        String text;
+
+        // Load Tasks from File
+        try {
+            FileManager.loadTasks(taskArrayList);
+        } catch (IOException e) {
+            System.out.println("ERROR OCCURRED.");
+        }
+
+        while(true) {
+
+            try {
+                Scanner in = new Scanner(System.in);
+                text = in.nextLine();
+
+                // "bye" -- terminate
+                if (text.equalsIgnoreCase("bye")) {
+                    break;
+                }
+
+                // "mark" -- mark task
+                else if (text.startsWith("mark")) {
+                    int position = Integer.parseInt(text.substring("mark".length() + 1)) - 1;
```

Comment:
can consider abstracting the command implementation for the different commands to the different command classes like in personsbook to make your main function more readable